CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+1.4.0 / 2019-08-28
+==================
+
+###### Changes
+* Dropping support for PHP 5.3
+* Updated documentation about testing
+* Syn library is [0.0.2](https://github.com/bitovi/syn/tree/v0.0.2)
+* `setDesiredCapabilities` will throw an exception if you will try to set capabilities on already started session
+* Added `getDesiredCapabilities` to allow fetch capabilities
+* Minors improvements to driver when running against W3C compatible driver
```

Comment:
Please move into `Bug fixes` sub-section and replace with:

```
* Fixed inability to manipulate windows when Selenium 3 with Firefox GeckoDriver was used
* The `clickOnElement` method wasn't working when Selenium 3 with Firefox GeckoDriver was used
```

Please add these into `Misc` sub-section:

```
* The `setDesiredCapabilities` method now combines default capabilities with default ones
* Removed outdated default capabilities
```

Please add these into `Bug fixes` sub-section:

```
* Don't use TABs, when triggering `change` JS event upon input value is change (fixes auto-complete control testing in Google Chrome)
```

Please add these into `New features` sub-section:

```
* Allow uploading files to remote Selenium instances (e.g. SauceLabs, BrowserStack, etc.)
```

---

After I've inspected all made commits I've noted couple of vital change log entries, that were omitted. Please pay attention next time.

Reply:
Thanks for help and understand. Package wasn't release for 3 years and because I'm not maintaining this package I'm not aware about all changes.

Updated.